packages/plugins/type-designer/src/components/elements-type-container/content-card.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang="ts">
+    import Card from "@smui/card";
+    import type { DataTypeTemplates } from "./data-type-templates";
+
+    export let elementType: DataTypeTemplates[];
+
+    function handleCardClick(item: string) {
+        console.log("Clicked:", item);
+    }
+</script>
+
+<section id="type-designer-card">
+    {#each elementType as elementType}
+        <Card
+            class="card"
+            on:click={() => handleCardClick(JSON.stringify(elementType))}
```

Comment:
why do we need to stringify it?

Reply:
DataTypeTemplates to string just to show it in the console. The handleCardClick function is set to be updated in a following ticket

Reply:
ok so we could also stringify in the `handleCardClick` too?

Reply:
No need for stringify at second glance:
dbbde8fb2a31b50aee7df55c4de8a9100dd5d58d